kiwi/builder/container.py

```diff
@@ -57,6 +60,23 @@ def __init__(self, xml_state, target_dir, root_dir):
         self.target_dir = target_dir
         self.container_config = xml_state.get_container_config()
         self.requested_container_type = xml_state.get_build_type_name()
+        self.base_image_uri = xml_state.get_derived_from_image_uri()
+        self.base_image = None
+
+        if self.base_image_uri:
+            base_image_origin = self.base_image_uri.translate()
+
+            # we expect the base image to be unpacked by the kiwi prepare step
+            # and stored inside of the root_dir/image directory
+            self.base_image = os.path.normpath(
+                os.sep.join(
+                    [
+                        root_dir, 'image',
+                        base_image_origin.replace('.tar.xz', '')
```

Comment:
@schaefi in my first commit that name was forced, since to my eyes this is something it should not depend on any user choice or configurable parameter, I named that file with a constant name (`image_file`) to avoid any kind of name assumption. Maybe I should have included some comment as it was not obvious reading the code why it was named like that. In my initial commit the file were forced by code to be called `root_dir/image/image_file`  and `root_dir/image/image_file.md5`, in build step I was just checking the `derived_from` attribute to validate if those files were expected to be found or not.

I think this could solve the naming assumption. What you think?